source/Generic/PlayState/PlayState.cs

```diff
@@ -576,52 +596,48 @@ private void ShowNotification(NotificationTypes status)
 
             if (settings.Settings.NotificationShowSessionPlaytime)
             {
-                sb.Append($"\n{ResourceProvider.GetString("LOCPlayState_Playtime")} {GetHoursString(GetRealPlaytime())}");
+                sb.Append($"\n{ResourceProvider.GetString("LOCPlayState_Playtime")} {GetHoursString(GetRealPlaytime(game))}");
```

Comment:
In here, it executes `GetRealPlaytime` method. This method then does
```csharp
            var gameData = GetGameData(game);
            if (gameData == null)
            {
                return 0;
            }
```
This method then gets called again for `if (settings.Settings.NotificationShowTotalPlaytime)`
You are already checking if the game has data at the start of the method, I think you should just pass that same data as argument to the required methods

Reply:
To prevent searching for this same data multiple times

Reply:
I see that it's also searched for in the `SwitchGameState()` method. If possible, please make it so it only searches once for it